include/p4mlir/Conversion/P4HIRToBMv2IR/Passes.h

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This fle contains the declarations to register conversion passes.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef P4MLIR_CONVERSION_P4HIRToBMv2IR_PASSES_H
+#define P4MLIR_CONVERSION_P4HIRToBMv2IR_PASSES_H
+
+// We explicitly do not use push / pop for diagnostic in
+// order to propagate pragma further on
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+
+#include "mlir/IR/DialectRegistry.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Pass/PassRegistry.h"
+
+#include <memory>
```

Comment:
Something strange with include order. Also, some things are included twice

Reply:
I re-run clang-format, should be ok now